### DISCUSSION TITLE
Graphical presentation challenge

### DISCUSSION THREAD

**Discussion by sengeruan** [msberends/AMR#89] [Q&A]
Dear the AMR Package Community,
Can anyone help me with this?
I was trying to select only my four bacterial isolates (from a column with different bacteria) with twenty three antibiotics columns and plot them. But this concept is failing to do what I want.  Additionally, I am thinking about how you derive the "antibiotic" argument  in the syntax, I am asking this because I was also trying to select only the antibiotics of interest rather than all in the dataset but also nothing comes as I wanted. 

You can take a look at my syntax to help me if possible to get the graph as the one below,

ggplot(my_antibiotic_dataset%>% group_by(bacterials_names %like% "E.coli", "klebsiella", "pseudomonas","proteus")) +
geom_rsi(x =  bacterials_names %like% "E.coli", "klebsiella", "pseudomonas","proteus" ) +
facet_rsi(facet = C(amikacin,amoxclav,ampicillin,cefazoline,cefepime, cefixime,cefotaxime,cefoxitin,ceftazidime,ceftriaxone, cephalexin,  chloramhenicol, ciprofloxacin,  doxycycline, gentamycin,  imepenem, levofloxacin,meropenem,nitrofurantoin,pepiracillin, tetracycline, tobramycin,  trimesulpha)) +
scale_rsi_colours() +
scale_y_percent(breaks = 0:4 * 25) +
  coord_flip() +
  labs(
    title = "Resistance per isolates and antibiotics",
    subtitle = "(Graphical presentation of Bacterial Resistance)") +
theme(axis.text.y = element_text(face = "italic"))

Your help will be much appreciated!
![image](https://user-images.githubusercontent.com/72040438/208236596-8b66d3cc-f97b-46b7-93c1-11ac9bf68e79.png)

**Comment by theanita1** [comment]
Would it be possible to make a subset with only the bacteria you want and then plot from there?
